networking/voting_handlers.go

```diff
@@ -340,7 +340,7 @@ func (s *Voting) PullQuery(validatorIDs ids.ShortSet, chainID ids.ID, requestID
 			peers = append(peers, peer)
 			s.log.Verbo("Sending a PullQuery to %s", vID)
 		} else {
-			s.log.Warn("Attempted to send a PullQuery message to a disconnected validator: %s", vID)
+			s.log.Warn("attempted to send a PullQuery message to a disconnected validator: %s", vID)
```

Comment:
This should be a Debug message (as well as other equivalent logs for different message types)